ERCS/erc-x.md

```diff
@@ -0,0 +1,238 @@
+---
+title: PermaLink Asset Bound Token Standard
+description: An interface for PermaLink Asset Bound Tokens, also known as a PermaLink-ABTs.
+author: Mihai Onila (@MihaiORO), Nick Zeman (@NickZCZ), Narcis Cotaie (@NarcisCRO)
+discussions-to: https://ethereum-magicians.org/t/non-fungible-asset-bound-token/23175
+status: Draft
+type: Standards Track
+category: ERC
+created: 2025-04-01
+requires: 721, 721Enumerable
+---
+
+### Abstract
+
+This standard introduces a subclass of tokens known as **PermaLink Asset Bound Tokens (PermaLink-ABTs)** a specific implementation of the broader **Asset Bound Token (ABT)** concept. ABTs establish a novel ownership paradigm where **an asset can own another asset**, enabling composable, nested, and portfolio-like token structures that evolve together over time.
+
+PermaLink-ABTs implement a permanent binding mechanism where a token in one smart contract is irreversibly linked to a token in another contract. These links mirror key state data such as `ownerOf`, `tokenId`, `totalSupply`, and `balanceOf` using the `assetBoundContract` interface. Traditional token transfer and approval functions are disabled to enforce immutability and structural cohesion between bound assets.
```

Comment:
```suggestion
PermaLink-ABTs implement a permanent binding mechanism where a token in one smart contract is irreversibly linked to a token in another contract. These links mirror key state data such as `ownerOf`, `tokenId`, `totalSupply`, and `balanceOf` using the `assetBoundContract` interface. Traditional token transfer and approval functions are omitted to enforce immutability and structural cohesion between bound assets.
```

Saying "omitted" is a bit more clear, since you don't actually include `transfer` (and friends) in your interface above.